uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1547,69 +1573,75 @@ sub CreateMRICandidates {
     ############################################################
     ### Standardize sex (DICOM uses M/F, DB uses Male/Female)
     ############################################################
-    my $this = shift;
-    my $query = '';
-    my ($subjectIDsref,$sex,$tarchiveInfo,$User,$centerID, $upload_id) = @_;
-    my ($message);
+    my $this  = shift;
+    my ($subjectIDsref, $tarchiveInfo, $User, $centerID, $upload_id) = @_;
+
+    my ($sex, $query, $message);
+    my $dbh    = $this->{dbhr};
```

Comment:
Just to avoid confusion, I would rename that variable into `$dbhr`

Reply:
ok. Done in the next commit.